app/grandchallenge/challenges/views.py

```diff
@@ -250,11 +247,9 @@ class ChallengeRequestBudgetUpdate(
     permission_required = "challenges.change_challengerequest"
     template_name = "challenges/challengerequest_budget_form.html"
 
+    @htmx_refresh
```

Comment:
This design doesn't feel right. Everywhere else in the codebase you need to return a response for `form_valid`, so when reading the code this looks like it breaks the Django API. As a developer I would be confused as to when to return a response or not. Also it's awkward to use in situations like this where you don't want to override `form_valid` but need to do this dance to use a decorator.

Would a mixin or a specialised `HTMXUpdateView` be more appropriate?